src/console/App.js

```diff
@@ -73,10 +98,10 @@ class App extends Component {
             </NavLink>
             <NavLink to="/shield">SHIELD</NavLink>
 
-            {(this.state.userInfo && (
-              <Button onClick={this.onUserLogout}>{`Logged in as ${
-                this.state.userInfo.nickname
-              }`}</Button>
+            {(this.props.userInfo && (
```

Comment:
Can we break this out into a `CurrentUserInfo` component or something? This `<button /> || <button />` will only continue to grow as we add more info/links to it, and it's already a little weird to parse (due to prettier's formatting).